src/field_theory/polynomial_galois_group.lean

```diff
@@ -324,6 +334,91 @@ begin
   rw [aeval_def, map_root_of_splits _ (splitting_field.splits p) hp],
 end
 
+lemma splits_ℚ_ℂ {p : polynomial ℚ} : fact (p.splits (algebra_map ℚ ℂ)) :=
+⟨is_alg_closed.splits_codomain p⟩
+
+local attribute [instance] splits_ℚ_ℂ
+
+/-- The number of complex roots equals the number of real roots plus
+  the number of roots not fixed by complex conjugation -/
+lemma gal_action_hom_bijective_of_prime_degree_aux {p : polynomial ℚ} :
```

Comment:
I guess that in the future this can be generalized to fields `K` with `algebra K real` (and a scalar tower with `complex`).